src/gui/qgsmaptooledit.h

```diff
@@ -40,10 +40,18 @@ class GUI_EXPORT QgsMapToolEdit: public QgsMapTool
 
     /**
      * Returns default Z value
-     * Use for set Z coordinate to new vertex for 2.5d geometries
+     * Use for set Z coordinate to new vertex
      */
     double defaultZValue() const;
 
+    /**
+     * Returns default M value
+     * Use for set M coordinate to new vertex
```

Comment:
Actually I'm not sure of the grammar. Maybe `Used for setting M coordinate to new vertex.`?